components/camera/camera.go

```diff
@@ -70,15 +72,32 @@ type NamedImage struct {
 	SourceName string
 }
 
+// ImageMetadata contains useful information about returned image bytes such as its mimetype.
+type ImageMetadata struct {
+	MimeType string
```

Comment:
rather than making this a string, what would you think about having this be an enum which has the same values as camerapb.Format https://github.com/viamrobotics/api/blob/main/proto/viam/component/camera/v1/camera.proto#L90

That way it is clear which specific mime_types we support.

Reply:
actually, never mind I see that the rest of the camera proto uses strings for mime type